drivers/clickhouse/clickhouse_test.go

```diff
@@ -0,0 +1,115 @@
+//go:build clickhouse
+
+package clickhouse
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/k1LoW/tbls/schema"
+	"github.com/stretchr/testify/assert"
+	"github.com/xo/dburl"
+	"os"
+	"testing"
+)
+
+var db *sql.DB
+
+const schemaName = "testdb"
+
+func TestMain(m *testing.M) {
+	var err error
+	db, err = dburl.Open("clickhouse://default@localhost:9000/testdb")
+	if err != nil {
+		fmt.Printf("%v\n", err)
+		os.Exit(1)
+	}
+	defer db.Close()
+	exit := m.Run()
+	if exit != 0 {
+		os.Exit(exit)
+	}
+}
+
+func TestInfo(t *testing.T) {
+	driver := New(db)
+	d, err := driver.Info()
+	assert.NoError(t, err)
```

Comment:
This is my preference, I would prefer to use the normal testing package and go-cmp instead of testify.

Reply:
I've removed [stretchr/testify](https://github.com/stretchr/testify) again.